WordPress/gutenberg
# [RNMobile] Bump iOS minimum deployment target to iOS 13.0

## Description

gutenberg-mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2876

- Bumped min deployment target to iOS 13
- Removed code specific to iOS versions before 13

## How has this been tested?




- Check if editor is launching
- Check if unsupported block editor is launching correctly as a modal and has correct background color

## Screenshots 
N/A

## Types of changes



Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
